src/regression_tests/regression_test.test.ts

```diff
@@ -0,0 +1,138 @@
+import fetchMock from 'jest-fetch-mock';
+import {
+  createUser,
+  getBalance,
+  getEnvVariables,
+  getPaymailDomain,
+  getTransactions,
+  RegressionTestConfig,
+  RegressionTestUser,
+  removeRegisteredPaymail,
+  sendFunds,
+} from './utils';
+
+const MINIMAL_FUNDS_PER_TRANSACTION = 2;
+// Two minute timeout for transaction tests
+const TEST_TIMEOUT_MS = 120000;
+const ADMIN_XPRIV =
+  'xprv9s21ZrQH143K3CbJXirfrtpLvhT3Vgusdo8coBritQ3rcS7Jy7sxWhatuxG5h2y1Cqj8FKmPp69536gmjYRpfga2MJdsGyBsnB12E19CESK';
+const ADMIN_XPUB =
+  'xpub661MyMwAqRbcFgfmdkPgE2m5UjHXu9dj124DbaGLSjaqVESTWfCD4VuNmEbVPkbYLCkykwVZvmA8Pbf8884TQr1FgdG2nPoHR8aB36YdDQh';
+
+let paymailDomainInstanceOne: string = '';
+let paymailDomainInstanceTwo: string = '';
```

Comment:
unnecessary explicit typing (`: string`) - this is deduced by the default value `= ''`

Reply:
Removed